src/main/java/com/senecafoundation/Scene/Scenario.java

```diff
@@ -1,37 +1,43 @@
 package com.senecafoundation.Scene;
 
-import java.util.Scanner;
+import java.util.List;
+import java.util.UUID;
 
-public class Scenario extends Choices
+public class Scenario
 {
     //Variables
-
-    private int id;
-
-    //Add another variable ---> private int nextSceneID and maybe previousSceneID
-
+    private String id;
+    private String nextSceneID;
+    private String previousSceneID;
     private String sceneName;
-    
     private String sceneText;
-
-    private Scanner input = new Scanner(System.in);
+    private List<Choice> choices;
```

Comment:
One of the things I think this team should spend more time understanding is the difference between "inherits from" another object - which is saying I have all your behaviors and I'm growing more on top of you, and...

_Depends on_ another object i.e. I need to use your features, but I'm not related to you or growing you.  In this case, a scenario depends on choices, and choices depend on responses, but they are not parent/children.  I.e. scenario isn't an inherited version of choice.  But it does need a property because it depends on choices.

inherits from vs. depends on - know the difference.